### ISSUE TITLE
Ads Credit Flow: Landing Page Update Details

### ISSUE THREAD

**Issue by budzanowski** [woocommerce/pinterest-for-woocommerce#535]
### Info

This is a supplement to #513.
Originally the #513 should also be concretised with proper copy link and image. For now they are placeholders. To unblock further development we move development of those items here. PR implementing #513 ( #525 ) can be merged.

Missing elements, currently present as placeholders, ( copy from #513 ):
 

- [x] Terms and conditions link 
- [x] Image 
- [x] Copy 
- [x] Final sign-of from design @shivanarrthine

**Comment by findingsimple** [comment]
On hold for confirmation of final copy/designs

**Comment by budzanowski** [comment]
Updated designs:
<img width="558" alt="image" src="https://user-images.githubusercontent.com/17271089/191460258-2ab7be3d-dcbb-4d40-b9e8-7d5af8071d37.png">

<img width="703" alt="image" src="https://user-images.githubusercontent.com/17271089/191460341-089d9ae0-97cf-4201-a38b-fa83eeba951a.png">
